src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -1925,7 +1925,8 @@ void W3DDisplay::Gather_Debug_Stats()
                 drawable->Get_Position()->y,
                 drawable->Get_Position()->z);
 
-            const PhysicsBehavior *phys = object->Get_Physics();
+            // #BUGFIX Test object before getting physics.
```

Comment:
We don't really add a comment like this when it's just a nullptr check

Reply:
It deviates from original, so comment should be ok, no?